raiden/transfer/mediated_transfer/events.py

```diff
@@ -671,3 +671,31 @@ def from_dict(cls, data: Dict[str, Any]) -> "EventUnexpectedSecretReveal":
         )
 
         return restored
+
+
+class EventRouteFailed(Event):
+    """ Event emitted when a route failed.
+
+    A route fails, when a RefundTransfer reaches the initator.
+    """
+
+    def __init__(self, secrethash: SecretHash):
+        self.secrethash = secrethash
+
+    def __repr__(self):
+        return f"<" f"EventRouteFailed " f"secrethash:{pex(self.secrethash)} " f">"
```

Comment:
uhhh, black messed up badly here.

Reply:
Oh damn, wonder what that would have done :)